packet/flatcar-linux/kubernetes/workers/cl/worker.yaml.tmpl

```diff
@@ -138,10 +138,20 @@ storage:
             # Don't mount if no RAID is configured.
             [ -z "$${raid_config}" ] && return 0
 
-            # Make mount persistent across reboots.
-            local devname=$(awk '{print $2}' /etc/mdadm.conf)
-            mount "$${devname}" /mnt/
-            echo "$${devname} /mnt ext4 defaults,nofail,discard 0 0" | tee -a /etc/fstab
+            cat /etc/mdadm.conf | while read line; do
+              # Make mount persistent across reboots.
+              local devname=$(echo "$${line}" | awk '{print $2}')
+
+              # Mount the RAID device only if there is a filesystem on the device
+              local fs=$(blkid "$${devname}")
+              [ -z "$${fs}" ] && continue
+
+              local mount_point=$(echo "$${devname}" | cut -d"/" -f 4)
```

Comment:
Isn't this the same as `basename` command?

Reply:
yep this is, thank you. TIL :slightly_smiling_face: